app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class QuotesController < ApplicationController
+  def index
```

Comment:
See my comments in your facade. If you were to implement something as I explained, your controller could look something like:
```
def index
  @search_phrase = params[:query]
  @total_found, @quotes = QuoteFacade.quote_search(@search_phrase)
end
```
Now you have access to display `@total_found` for the total count and `@quotes` for the quote attributes.